### DISCUSSION TITLE
How to run `esbuild --watch` using turborepo?

### DISCUSSION THREAD

**Discussion by thasmo** [vercel/turborepo#1337] [Help] [answered]
When running `turbo run dev --no-cache --parallel` or similar in the root of the project, the `dev` tasks in all workspace packages, defined as `esbuild index.ts --watch ...` exit and do not keep running.

What's the correct way to run all packages' `dev` task (`esbuild index.ts --watch ...`) using turborepo?

Thanks a lot!

**Comment by mehulkar** [comment, marked as answer]
`turbo run dev --parallel` should have worked. If this is still relevant for you, could you open an issue and show the output with the `-vvv` flags?
